src/components/Pollution.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { airpollution } from '../Redux/Reducers/Pollution';
+import '../App';
+
+const Time = (element) => {
+  const date = new Date(element.props * 1000);
+  return `${date.toLocaleDateString()}\n${date.toLocaleTimeString()}`;
+};
+
+const Pollution = () => {
+  const location = useLocation();
+  const dispatch = useDispatch();
+  const info = location.state;
+  useEffect(() => {
+    dispatch(airpollution(info));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
As mentioned earlier and as requested by the previous reviewer, kindly avoid disabling linter checks. You may read more about the [react-hooks/exhaustive-deps](https://blog.logrocket.com/understanding-react-exhaustive-deps-linting-warning/).